src/trello2md.py

```diff
@@ -177,26 +184,32 @@ def main():
                     markdown.append(print_checklists(card['id'], data))
 
     # save result to file
-    if (args.output):
-        outputfile = args.output
+    to_stdout = False
+    if (args.to_stdout):
+      to_stdout = True 
     else:
-        outputfile = args.inputfile.replace('.json', '.md')
-        if outputfile == args.inputfile:
-            outputfile += '.md'
+      if (args.output):
+          outputfile = args.output
+      else:
+          outputfile = args.inputfile.replace('.json', '.md')
+          if outputfile == args.inputfile:
+              outputfile += '.md'
 
     try:
-        with open(outputfile, 'w', encoding = args.encoding) as of:
+        if(not to_stdout):
+          with open(outputfile, 'w', encoding = args.encoding) as of:
             of.write(''.join(markdown))
+          print('Sucessfully translated to "{0}"!'.format(outputfile))
 
-        print('Sucessfully translated to "{0}"!'.format(outputfile))
-
-        if args.card_links:
-            print('Option --card-links is currently unimplemented and ignored.')
+        else:
+          markdownText = ''.join(markdown)
+          sys.stdout.reconfigure(encoding='utf-8')
```

Comment:
I don't know how `stdout.reconfigure` works.  What if the specified input encoding is not UTF-8? Does it sense to use `args.encoding` here?

Reply:
I know it's not PEP-8, but for consistency with the rest:

```suggestion
          sys.stdout.reconfigure(encoding = 'utf-8')
```

Reply:
(Or you might as well remove all spaces around `=` in keyword arguments.)